exo-setup/features/setup.feature

```diff
@@ -20,14 +21,77 @@ Feature: Setup of Exosphere applications
       | dashboard/Dockerfile     |
       | mongo-service/Dockerfile |
       | web-server/Dockerfile    |
+    And it has generated the file "docker-compose.yml" with the content:
+      """
+      version: '3'
```

Comment:
I'll put it in a tmp dir. This config file should be accessible to the user for reading but it shouldn't be modified nor checked into GitHub.